src/cc/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set_target_properties(bcc-static PROPERTIES OUTPUT_NAME bcc)
 # BPF is still experimental otherwise it should be available
 #llvm_map_components_to_libnames(llvm_libs bpf mcjit irreader passes)
 llvm_map_components_to_libnames(llvm_libs bitwriter bpfcodegen irreader linker
-  mcjit objcarcopts option passes x86codegen)
+  mcjit objcarcopts option passes x86codegen powerpccodegen)
```

Comment:
what's the difference in .so size after inclusion of ppc backend?
is it really needed?
can we select x86codegen or powerpccodegen at build time?
I think we should try to remove x86. We can probably just stub LLVMInitializeX86Target*